queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals where date_of_birth BETWEEN DATE '2016-01-01' AND '2019-01-01';
+SELECT name, neutered, escape_attempts FROM  animals WHERE neutered = 'yes' AND escape_attempts < 3;
```

Comment:
- [ ] This query is about selecting names only of all animals that are neutered and have less than 3 escape attempts please update it

Reply:
I changed to list only names in this query!